protractor.conf.js

```diff
@@ -43,6 +43,7 @@ exports.config = {
   },
 
   onPrepare: () => {
+    require('ts-node').register({ project: 'e2e' });
     browser.driver.manage().window().setSize(1920, 1080);
     let SpecReporter = require('jasmine-spec-reporter').SpecReporter;
 
```

Comment:
pay attention to selenium promise manager https://github.com/angular/protractor/tree/5.1.2/exampleTypescript/asyncAwait